### ISSUE TITLE
[DOC] Improve Setting the Row Group Size for the Parquet Files section in docs

### ISSUE THREAD

**Issue by rnyak** [NVIDIA-Merlin/NVTabular#1564]
## Report incorrect documentation

**Location of incorrect documentation**

We have [troubleshooting])(https://nvidia-merlin.github.io/NVTabular/main/resources/troubleshooting.html#setting-the-row-group-size-for-the-parquet-files) section in NVT docs page to guide users to avoid memory issues when running NVT or even training a model. However we need more functionality and clarification [Setting the Row Group Size for the Parquet Files] (https://nvidia-merlin.github.io/NVTabular/main/resources/troubleshooting.html#setting-the-row-group-size-for-the-parquet-files) section.

Please see below for suggestions.

**Describe the problems or issues found in the documentation**

1. It is not clear how a user can set `row_group_size` when saving processed parquet files from NVT to disk. Currently we do not have `row_group_size` arg to set in the `to_parquet()` method. What'd be the workaround for that? Better to provide a simple example script.

2. We can add how one can checkout memory size of each partition something like that:

```
import dask_cudf
for part in dask_cudf.read_parquet(PATH, split_row_groups=1).partitions:
    print(part.memory_usage(index=True).sum().compute() / 1e9, "GB")
```

3. Also, it'd be better to explain if `num_of_row_groups` and `num_partitions` refer to same thing, if, not explain the reason why. This is unclear.

**Comment by rnyak** [comment]
@rjzamora does this DOC enhancement make sense to you? thanks.

**Comment by rjzamora** [comment]
>does this DOC enhancement make sense to you?

Yes. It does look like the "Setting the Row Group Size for the Parquet Files" section is wrong: The cudf argument is different than pandas/pyarrow (`row_group_size_rows` vs `row_group_size`).

Note that I'd also like to add a `row_group_size_rows` option to `Dataset`, but https://github.com/rapidsai/cudf/issues/10978 will need to be resolved first to support the cudf backend.
